packages/react-native/tsconfig.json

```diff
@@ -1,12 +1,13 @@
 {
   "extends": "@posthog-tooling/tsconfig-base",
   "compilerOptions": {
+    "rootDir": ".",
     "declaration": true,
     "esModuleInterop": true,
     "jsx": "react-native",
     "lib": ["ESNext"],
     "module": "ESNext",
-    "moduleResolution": "node16",
+    "moduleResolution": "bundler",
```

Comment:
we need to be sure this is not breaking anything
https://reactnative.dev/blog/2023/06/21/package-exports-support#enabling-package-exports-beta
i recall changing [something around this for a reason](https://github.com/PostHog/posthog-js-lite/pull/294/files#r1815663690), maybe this is ok, but it has to be tested